tests/test_examples/test_runs_via_configs/test_via_configs.py

```diff
@@ -59,3 +63,99 @@ def test_train_with_categories(accelerator: str, devices: int) -> None:
 @pytest.mark.parametrize("accelerator, devices", accelerator_devices_pairs())
 def test_val(accelerator: str, devices: int) -> None:
     run("val_mock.py", accelerator, devices)
+
+
+@pytest.mark.parametrize("accelerator, devices", accelerator_devices_pairs())
+def test_inference_error_two_sources_provided(accelerator: str, devices: int) -> None:
+    try:
+        run("inference_images_mock.py", accelerator, devices)
+    except subprocess.CalledProcessError:
+        pass
+    else:
+        raise AssertionError("This test should raise exeption for two input sources provided")
+
+
+@pytest.fixture()
+def features_from_path_command_and_paths(
+    accelerator: str, devices: int, cleanup: bool = True
+) -> Iterator[Tuple[str, Path, Path]]:
+    features_path: Path = MOCK_DATASET_PATH / "features1.json"
+    command = (
+        f"inference_images_mock.py ~dataframe_name dataset_root={MOCK_DATASET_PATH} " f"features_file={features_path}"
+    )
+    cfg_path = SCRIPTS_PATH / "configs" / "inference_images_mock.yaml"
+    yield command, features_path, cfg_path
+    features_path.unlink()
+
+
+@pytest.fixture()
+def features_from_df_command_and_paths(
```

Comment:
I feel, that the names `params_for_inference_on_df` and `params_for_inference_on_folder` are better. What do you think?

Reply:
It also helps you to avoid using `vairable_..._path_path` below

Reply:
I agree, will do that